src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -11,7 +11,7 @@
 import org.myrobotlab.framework.ServiceType;
 import org.myrobotlab.framework.Status;
 import org.myrobotlab.framework.interfaces.ServiceInterface;
-import org.myrobotlab.io.FileIO;
+import org.myrobotlab.inmoov.Utils;
```

Comment:
Why the need for an additional utils class?

Reply:
Because loadFile with be used for other things that loadGesture, also this will reduce codes lines in the high level

Reply:
if it's used by something other than InMoov why is it in the inmoov package?  Also, at the end of the day, this method just calls python.exec ..

Reply:
something other than Gestures but still related to inmoov. There is some stuffs inside python land not yet pushed here like language pack ( translated vars )